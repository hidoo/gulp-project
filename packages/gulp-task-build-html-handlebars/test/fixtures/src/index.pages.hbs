---
seo:
  title: "page title"
  description: "page description."
---
{{#each @root.pages}}
- - -
title: {{this.seo.title}}
description: {{this.seo.description}}
path: {{this.path.relative}}
{{/each}}
