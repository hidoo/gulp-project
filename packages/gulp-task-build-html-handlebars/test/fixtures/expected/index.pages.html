title: page title | site title
description: page description.site description.
title: page title
description: page description.
title: page title
description: page description.
title: page title
description: page description.
title: page title
description: page description.
